openfisca_core/formula_toolbox.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not fond of the naming `formula_toolbox` (like other variants like `_helpers` or `_functions`).

I would prefer to name the module after what's inside.

Second, this module contains only imports, which is weird. I would prefer to import directly these modules from the impacted countries (like [here](https://github.com/openfisca/openfisca-france/pull/697/files#diff-5d19a690800f810691075aef479a06afR3))

But this is transitive and does not stop there! I mean `openfisca_france/model/base.py` should not exist neither lines like `from openfisca_france.model.base import *`.

We've got to do a choice: either we keep these "forwarding-only modules", either we forbid them and make *real* imports from modules.